lib/dal-test/src/lib.rs

```diff
@@ -538,7 +538,7 @@ async fn global_setup(test_context_builer: TestContextBuilder) -> Result<()> {
 
     info!("creating builtins");
     // TODO: @stack72 - remove this code path and install these from the module-index??
-    dal::migrate_builtins(
```

Comment:
Only local builtin schemas are migrated. This should make it a bit more obvious when traversing `global_setup` code.